app/page.tsx

```diff
@@ -1,20 +1,134 @@
+'use client'
+import useSWR from 'swr'
 import HeroSection from './components/HeroSection/HeroSection'
 import MovieList from './components/MovieList/MovieList'
-import { buttonsList, moviesList } from './mockdata'
+import { API, baseURL, fetcher, imgURL } from './api/api'
+import { useState } from 'react'
 
 export default function Home() {
+  /* Trending Movies Section*/
+  const [time, setTime] = useState<'day' | 'week'>('day')
+
+  const {
+    data: trendingMoviesData,
+    error: trendingMoviesError,
+    isLoading: trendingMoviesIsLoading,
+  } = useSWR(`${baseURL}/3/trending/movie/${time}?api_key=${API}`, fetcher)
+
+  const {
+    data: popularMoviesData,
+    error: popularMoviesError,
+    isLoading: popularMoviesIsLoading,
+  } = useSWR(`${baseURL}/3/movie/popular?api_key=${API}`, fetcher)
+
+  if (trendingMoviesError) return <div>failed to load</div>
+  if (trendingMoviesIsLoading) return <div>loading...</div>
+
+  const trendingMovies = trendingMoviesData.results.map((results: any) => {
+    const { poster_path, release_date, title, vote_average } = results
+    return {
+      cardImage: {
+        imgSrc: `${imgURL}${poster_path}`,
+        imgAlt: 'not found',
+      },
+      rating: {
+        value: Math.floor(vote_average * 10),
+      },
+      title: title,
+      date: release_date,
+      menuPopOverLinks: {
+        links: [
+          {
+            text: '1',
+            url: '#',
+          },
+          {
+            text: '1',
+            url: '#',
+          },
+          {
+            text: '1',
+            url: '#',
+          },
+        ],
+      },
+    }
+  })
+
+  /* Polular Movies */
+
+  if (popularMoviesError) return <div>failed to load</div>
+  if (popularMoviesIsLoading) return <div>loading...</div>
```

Comment:
we can combine two similar messages like:
if(popularMoviewError || trendingMoviesError){
return <p>Loading...</p>
}